notebooks/00_Setup.ipynb

```diff
@@ -47,8 +47,18 @@
    "id": "aa6bddcb",
```

Comment:
<p>Can you mention installing GPU supported jax via conda: <a href="https://jax.readthedocs.io/en/latest/installation.html#conda-installation" target="_blank">https://jax.readthedocs.io/en/latest/installation.html#conda-installation</a>?</p><p> </p><p>This is for decoding. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass/pull/1029/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>